Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -3225,6 +3350,107 @@ def get_script_execution_result_files_command(client: Client, args: Dict) -> Dic
     return fileResult(filename, file_response.content)
 
 
+def decode_dict_values(dict_to_decode: dict):
+    """Decode JSON str values of a given dict.
+
+    Args:
+      dict_to_decode (dict): The dict to decode.
+
+    """
+    for key, value in dict_to_decode.items():
+        # if value is a dictionary, we want to recursively decode it's values
+        if isinstance(value, dict):
+            decode_dict_values(value)
+        # if value is a string, we want to try to decode it, if it cannot be decoded, we will move on.
+        elif isinstance(value, str):
+            try:
+                dict_to_decode[key] = json.loads(value)
+            except ValueError:
+                continue
+
+
+def filter_general_fields(alert: dict) -> dict:
+    """filter only relevant general fields from a given alert.
+
+    Args:
+      alert (dict): The alert to filter
+
+    Returns:
+      dict: The filtered alert
+    """
+
+    updated_alert = {}
+    updated_event = {}
+    for field in ALERT_GENERAL_FIELDS:
+        if field in alert:
+            updated_alert[field] = alert.get(field)
+
+    event = alert.get('raw_abioc', {}).get('event', {})
+    if not event:
+        return_warning('No XDR cloud analytics event.')
+        sys.exit(0)
```

Comment:
are you sure we want to stop? what about the rest of the list? shouldn't it just be skipped and the rest get processed?
```suggestion
        return_warning('Not an XDR cloud analytics event.', exit=True)
```